metricbeat/docs/modules/aerospike.asciidoc

```diff
@@ -25,7 +25,6 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 ----
 metricbeat.modules:
 - module: aerospike
-  metricsets: ["namespace"]
```

Comment:
I removed the metricset config from the short config to make it even shorter, but not sure yet if this is a good idea.

Reply:
Working on further modules I realised we should make a decision for each module. If there is only 1 metricset for example, we don't need it. Updated PR accordingly.